core/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -71,15 +74,18 @@
 
     private ShardSearchFailure[] shardFailures;
 
+    private Clusters clusters;
```

Comment:
can be final?

Reply:
I'm afraid not, SearchResponse implements Streamable and not Writeable as it is an ActionResponse.

Reply:
fair enough